EventFilter/CTPPSRawToDigi/interface/CTPPSPixelRawToDigi.h

```diff
@@ -0,0 +1,47 @@
+#ifndef CTPPSPixelRawToDigi_H
+#define CTPPSPixelRawToDigi_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h